app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
```

Comment:
Ah ok. Yes. Maybe we can apply all of this to the ilios-chart component. I'm thinking:
1) Get rid of width / height properties on all chart components
2) Handle size through CSS completely
3) Keep isIcon and just apply that as a class on the ilios-chart component.

Will that work with the D3 svg? I'm basically just wishing and making stuff up so let me know if that wont work at all.